Tests/Runner/BackendAssertsTrait.php

```diff
@@ -21,4 +21,18 @@ public static function assertFileBackend(string $expectedProvider, Client $clien
             ))
         );
     }
+
+    public static function assertDefaultTableBackend(string $expectedBackend, Client $client): void
+    {
+        $owner = $client->verifyToken()['owner'];
+        assert(
+            $owner['defaultBackend'] === $expectedBackend
+                && $owner[sprintf('has%s', mb_convert_case($expectedBackend, MB_CASE_TITLE))] === true,
+            new RuntimeException(sprintf(
+                'Project "%s" is not configured with %s table storage backend',
```

Comment:
Tak jsem zvědavaj za jak dlouho se to něčím rozmrdne :)

Reply:
Imho ne